backend/app/views/spree/admin/shipping_methods/_form.html.erb

```diff
@@ -33,6 +33,24 @@
     </div>
   </div>
 
+  <div class="alpha omega twelve columns">
+    <div class="alpha six columns">
+      <%= f.field_container :carrier do %>
+        <%= f.label :carrier, Spree.t(:carrier) %><br />
```

Comment:
Please remove the translation here and let rails do it's work. 

I know, that the other form fields still has the unnecessary translations in their labels and this is out of scope here, but we should not introduce even more too generic and context-less translation keys.